day_7/section1To4.md

```diff
@@ -0,0 +1,134 @@
+# Section 1: Super Learners
+
+### Reflect:
+
+1. What do you already do?
+  - To exercise my mind I usually play games. I feel like while I’m playing games I’m really focused on small details, my mind comes up with different ideas of playing. I know I’m not improving my learning skill much as I’d if I’m reading books but it helps me clear my mind. When I learn something new and interesting I usually share with my friends as well. I love reading about different fun facts and share them with my friends.
+
+2.  Does this make you want to do anything new or different as you begin this learning journey?
+  - Yes, it does. Knowing most of the successful people reading so many books makes me want to get into habit of reading books as well. If I read books I believe I will be able to improve my English grammar and vocabulary as well.
+
+
+# Section 2 - Asking Questions
+
+### Reflect:
+
+1. How you feel about asking questions? (specifically, the technical question as you enter your technical work at Turing)
+  - When I don't know the team well I hesitate to ask questions at first. I'm still learning English, I get worried if I'm going to hold the entire class when I ask questions.  Because if I'm not able to clearly ask questions then we will spend half of the time trying to figure out what I'm trying to ask. After I feel comfortable with the team, I usually find myself asking lots of questions. This is the first time I have ever felt so passionate about studying so, I'm sure I will be asking lots of questions during my time here at Turing. I've been learning different coding terms to help me able to ask questions clearly as well.
```

Comment:
Do NOT worry about holding anyone back or asking questions! We've worked with many students who aren't native English speakers and they have been very successful in our program. However, you'll need to be proactive in asking for help or asking your instructors to explain things differently. Trust me, your cohortmates will appreciate it too - asking questions can lead to much better understanding of concepts!